packages/database-mongo-sessions/src/mongo-sessions.ts

```diff
@@ -124,7 +124,9 @@ export class MongoSessions implements DatabaseInterfaceSessions {
 
     const ret = await this.sessionCollection.insertOne(session);
     // keep ret.ops for compatibility with MongoDB 3.X, version 4.X uses insertedId
-    return ((ret.insertedId ?? ret.ops[0]._id) as ObjectID).toString();
+    // return ((ret.insertedId ?? ret.ops[0]._id) as ObjectID).toString();
```

Comment:
can you please remove the commented line?

Reply:
Done.  [c669c93](https://github.com/accounts-js/accounts/pull/1251/commits/c669c93019aeb9385b596c6388d47d5ec27f0c86)